workflow-testing/src/main/java/com/squareup/workflow/testing/RenderTester.kt

```diff
@@ -350,6 +433,58 @@ interface RenderTester<PropsT, StateT, OutputT, RenderingT> {
    * handle a workflow or worker output or a rendering event.
    */
   fun render(block: (rendering: RenderingT) -> Unit = {}): RenderTestResult<PropsT, StateT, OutputT>
+
+  /**
+   * Describes a call to
+   * [RenderContext.renderChild][com.squareup.workflow.RenderContext.renderChild].
+   *
+   * @param workflow The child workflow that is being rendered.
+   * @param props The props value passed to `renderChild`.
+   * @param outputType The [KType] of the workflow's output type.
+   * @param renderingType The [KType] of the workflow's rendering type.
+   * @param renderKey The string key passed to `renderChild`.
+   */
+  class RenderChildInvocation(
+    val workflow: Workflow<*, *, *>,
+    val props: Any?,
+    val outputType: KTypeProjection,
+    val renderingType: KTypeProjection,
```

Comment:
These types are not trivial to pull out of a generic `Workflow` instance, and by providing them here we allow tests to provide default rendering values for any workflows that have a certain rendering type, without having to know the details about the specific workflows (e.g. don't require exact expectations for all child workflows that have `Unit` rendering types).

Reply:
Worth saying so in the kdoc?

Reply:
It does – there's a whole section about exactly this.